vlm/src/test/scala/geotrellis/contrib/vlm/gdal/GDALWarpOptionsSpec.scala

```diff
@@ -101,25 +98,23 @@ class GDALWarpOptionsSpec extends FunSpec with RasterMatchers with BetterRasterM
     }
 
     it("raster sources optimized transformations should behave in a same way as a single warp application") {
-      val base = GDAL.open(filePath)
+      val base = filePath
       val optimizedRawResample = dsresampleOpt(base)
-      val originalRawResample = dsresample(dsreproject(base), Some(filePath))
+      val originalRawResample = rasterSourceFromUriOptions(base, reprojectOptions.combine(resampleOptions))
 
       val rs =
         GDALRasterSource(filePath)
           .reproject(
-            targetCRS        = WebMercator,
+          targetCRS        = WebMercator,
             reprojectOptions = ReprojectOptions.DEFAULT.copy(targetCellSize = CellSize(10, 10).some),
             strategy         = AutoHigherResolution
-          )
-          .resampleToRegion(
-            region = RasterExtent(Extent(-8769160.0, 4257700.0, -8750630.0, 4274460.0), CellSize(22, 22))
-          )
```

Comment:
This is also kind of a 'stacking' operation happens here. Btw this particular example _can have a numerical mistake_ so feel free to fix it.

Reply:
The difference that I saw did not look numerical, my recollection is that cell size was entirely different.  It could be that that was not getting properly built-in to the warp options some where.

Reply:
Thanks, I understand what is happening now.

Reply:
Addressed.